platform/admin/core/admin-core-migration-commands/src/main/java/org/codice/ddf/migration/commands/ExportCommand.java

```diff
@@ -59,15 +63,15 @@ public ExportCommand(ConfigurationMigrationService configurationMigrationService
         this(configurationMigrationService, security, pathBuilder.build());
     }
 
-    ExportCommand(ConfigurationMigrationService configurationMigrationService,
-            Security security, Path path) {
+    ExportCommand(ConfigurationMigrationService configurationMigrationService, Security security,
```

Comment:
Oh, I see. It looks like in the Karaf commands code, they wrote setters for the `@Reference` variables that are only ever used in the unit tests (https://github.com/apache/karaf/blob/karaf-4.0.7/config/src/main/java/org/apache/karaf/config/command/ConfigCommandSupport.java#L56). Maybe we should do that instead so that the constructors are not confusing.